api/base/serializers.py

```diff
@@ -129,8 +174,8 @@ def to_representation(self, value):
         url = super(JSONAPIHyperlinkedIdentityField, self).to_representation(value)
 
         meta = {}
-        for key in self.meta or {}:
-            if key == 'count':
+        for key in self.meta:
+            if key == 'count' or key == 'unread_comments_count':
```

Comment:
Perhaps `if key in {'count', 'unread_comments_count'}`:` would be more concise? (trying to fight the urge on the trivial comments; sorry)